# generated by patch-package 6.5.0 on 2022-04-04 18:02:04
#
# command:
#   npx patch-package rpc-utils
#
# declared package:
#   rpc-utils: 0.4.0
#
diff --git a/node_modules/rpc-utils/dist/rpc-utils.cjs.development.js b/node_modules/rpc-utils/dist/rpc-utils.cjs.development.js
index 5a9584c..315bb20 100644
--- a/node_modules/rpc-utils/dist/rpc-utils.cjs.development.js
+++ b/node_modules/rpc-utils/dist/rpc-utils.cjs.development.js
@@ -556,13 +556,13 @@ var runtime = (function (exports) {
   // iterator prototype chain incorrectly implement this, causing the Generator
   // object to not be returned from this call. This ensures that doesn't happen.
   // See https://github.com/facebook/regenerator/issues/274 for more details.
-  Gp[iteratorSymbol] = function() {
+  define(Gp, iteratorSymbol, function () {
     return this;
-  };
+  });
 
-  Gp.toString = function() {
+  define(Gp, "toString", function () {
     return "[object Generator]";
-  };
+  });
 
   function pushTryEntry(locs) {
     var entry = { tryLoc: locs[0] };
diff --git a/node_modules/rpc-utils/dist/rpc-utils.esm.js b/node_modules/rpc-utils/dist/rpc-utils.esm.js
index e3a6532..da8db4a 100644
--- a/node_modules/rpc-utils/dist/rpc-utils.esm.js
+++ b/node_modules/rpc-utils/dist/rpc-utils.esm.js
@@ -552,13 +552,13 @@ var runtime = (function (exports) {
   // iterator prototype chain incorrectly implement this, causing the Generator
   // object to not be returned from this call. This ensures that doesn't happen.
   // See https://github.com/facebook/regenerator/issues/274 for more details.
-  Gp[iteratorSymbol] = function() {
+  define(Gp, iteratorSymbol, function () {
     return this;
-  };
+  });
 
-  Gp.toString = function() {
+  define(Gp, "toString", function () {
     return "[object Generator]";
-  };
+  });
 
   function pushTryEntry(locs) {
     var entry = { tryLoc: locs[0] };
